packages/extension-chakra-storefront/package.json

```diff
@@ -68,12 +68,12 @@
     "@loadable/component": "^5.15.3",
     "@peculiar/webcrypto": "^1.4.2",
     "@salesforce/cc-datacloud-typescript": "^1.0.10",
-    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.0",
+    "@salesforce/commerce-sdk-react": "^4.0.0-extensibility-preview.0",
```

Comment:
```suggestion
    "@salesforce/commerce-sdk-react": "^4.0.0-extensibility-preview.1",
```

Reply:
Actually, the `peerDependency` version must point to a version that is already exist on npm. 

Attempting to fix this caused CI to fail. This is a known issue with extensibility for a little while now and I don't want to fix that for this PR.